include/eve/function/rat.hpp

```diff
@@ -30,19 +30,21 @@ namespace eve
   //!
   //!  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~{.cpp}
   //!  auto operator()(floating_value auto x) const noexcept;
-  //!  template< floating_real_value T, floating_real_value S> auto operator()( T x, S tol) const noexcept;
+  //!  auto operator()( floating_value auto x, floating_value auto tol) const noexcept;
   //!  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   //!
   //! ---
   //!
   //! **Parameters**
   //!
   //!`x`:   [floating_real_value](@ref eve::value).
-  //!`tol`:   [floating_real_value](@ref eve::value).
+  //!`tol`: [floating_real_value](@ref eve::value) default to T(1.0e-6)*eve::abs(x).
```

Comment:
`tol`: [floating_real_value](@ref eve::value). By default, `tol` is equal to T(1.0e-6)*eve::abs(x).

Reply:
done